includes/wikia/services/ArticleQualityService.php

```diff
@@ -158,6 +159,18 @@ public function getArticleQuality() {
 			$article = new Article( $title );
 			$parser = $article->getParserOutput();
 
+			if ( !$parser ) {
+				//MAIN-3592
+				WikiaLogger::instance()->error(
+					__METHOD__,
+					[
+						'message' => 'Article::getParserOutput returned false',
```

Comment:
Method name will be avail in different field still readable. Do we need redundancy here?